Pull_Request_and_Code_Review_Standards.md

```diff
@@ -0,0 +1,49 @@
+
+-   Before marking the PR as "Ready for Review", contributors should also try their best to make sure that they are following the organization's [coding standards](https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CODING_STANDARDS.md) and to make sure that CI/CD pipelines are executing successfully. If this is not possible for some reason (e.g. because the code in *other repositories* must be changed) - the contributor should create new issue(s) for those required code changes, link them in the PR, and shortly explain why the change in the PR itself is not possible.
+
+-   Ideally, there should be a one-to-one mapping of issues to PR, although if a number of issues are related to the same underlying functionality, it is completely acceptable to address multiple issues in a single PR.
+
+-   If it's a PR that targets multiple issues or adds significant new functionality, there might be multiple rounds of code reviews and multiple tasks to be completed. In such cases, try to prepare a checklist in the PR description and keep it updated so that the reviewers and contributors always have an up-to-date view of the status of the PR.
+
+-   Reviewers should do their due diligence to understand the underlying issue, instead of asking contributors to reiterate the details that have already been outlined in GitHub Issues/Discussions.
+
+-   Reviewers should refrain from rubber-stamping reviews. If they feel unqualified or do not have time to review, they should urge contributors to request review from other collaborators instead of rubber-stamping an approval.
+
+-   Without the express permission or request of the contributor, the reviewers/collaborators should **not** directly push in the PR.
+
+-   If reviewers/contributors missed the issue that motivated the PR and would like to discuss it further when they see the PR, the discussion should take place in the original *issue* and not in the *PR* to minimize cognitive overhead of tracking the same discussion in two different places and to minimize the possibility of talking past each other.
+
+-   The responsibility of resolving code review comments falls squarely on the reviewers. The contributors shouldn't themselves resolve the review comments, even if they are 100% confident that their latest commits address review comments.
+
+-   Reviewers should not forget to review test code to make sure that it follows organization's coding standards.
+
+-   Code review comments should strictly follow the ethical decorum stipulated by [Code of Conduct](https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CODE_OF_CONDUCT.md).
+
+-   Reviewers should refrain from making remarks that can potentially be construed as hurtful or dismissive by contributors (e.g. "LOL, this doesn't work", "LMAO, what the hell is this?!", etc.). The comments should be constructive, irrespective of whether they are terse or detailed.
```

Comment:
LOL, or  this does not work is absolutely acceptable. This is not harmful or dismissive. Sometimes, this is just a funny comment, or the code is clearly wrong.
Let's try to have fun here.

Reply:
> This is not harmful or dismissive.

A reviewer might _think_ that what they are saying is not harmful or dismissive, but can they guarantee that it would not be _received_ as such?